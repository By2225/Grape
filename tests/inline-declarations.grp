fun Int main() {
        prints("HI DUDE!");

        Int a = 5;

        print(a);

        String b = "Hi People!";

        prints(b);

        return a;
}
