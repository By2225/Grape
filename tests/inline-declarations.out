././scripts/test: line 11: ././tests/inline-declarations: No such file or directory
